pkg/download/latest.go

```diff
@@ -6,26 +6,27 @@ import (
 	"github.com/gobuffalo/buffalo"
 	"github.com/gobuffalo/buffalo/render"
 	"github.com/gomods/athens/pkg/errors"
-	"github.com/gomods/athens/pkg/log"
 	"github.com/gomods/athens/pkg/paths"
 )
 
 // PathLatest URL.
 const PathLatest = "/{module:.+}/@latest"
 
 // LatestHandler implements GET baseURL/module/@latest
-func LatestHandler(dp Protocol, lggr log.Entry, eng *render.Engine) buffalo.Handler {
+func LatestHandler(dp Protocol, eng *render.Engine) buffalo.Handler {
 	const op errors.Op = "download.LatestHandler"
 	return func(c buffalo.Context) error {
 		mod, err := paths.GetModule(c)
 		if err != nil {
-			lggr.SystemErr(errors.E(op, err))
+			c.Logger().Warn(errors.E(op, err))
```

Comment:
This won't work because `Warn` is a hard coded. How do we deduce the log level based on the error.Error struct with this?

Reply:
Agree. If we use the level explicitly instead of relying on the error we are stepping back from what we have right now.

Reply:
what do you think of a `SystemErr` function in `github.com/gomods/athens/pkg/log` to do this?

Reply:
I think it would work. It's what you proposed down here https://github.com/gomods/athens/pull/645/files#diff-7522a4893602f93e19a3c36e780166e2R36